### PR TITLE
1 setup environment

### PR DESCRIPTION
**Pull request by marrakonda**
That PR is focused on environment setup:

- Dockerfile
- Github related stuff
- Basic Bot setup

All bot abilities (receiving parameters, slashes, working with button, images) will be addressed in the separate PR.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (sche) [inline .github/ISSUE_TEMPLATE/---bug-report.md:1]: 👍🏻

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.9
```
Comment (sche) [inline Dockerfile:1]: I think 3.11 is safe to use. 3.9 is a bit too old already

Comment (marrakonda) [reply Dockerfile]: Done

**Comment by marrakonda** [comment]
@sche All bot abilities (receiving parameters, slashes, working with button, images) will be addressed in the separate PR if it fine for you. After that PR s merged I will checkmark all items in the issue.

**Review by sche — APPROVED** [review]

**Comment by sche** [comment]
Good to go. Nice work 👏

**Pull request merged**